packages/apps/app-dashboard/src/components/user-dashboard/landing/ConnectToVincentYieldModal.tsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from 'react';
+import { IRelayPKP } from '@lit-protocol/types';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/shared/ui/dialog';
+import { Button } from '@/components/shared/ui/button';
+import { theme } from '../connect/ui/theme';
+import { AlertCircle, ArrowRight } from 'lucide-react';
+
+interface ConnectToVincentYieldModalProps {
+  agentPKP: IRelayPKP;
+}
+
+export function ConnectToVincentYieldModal({ agentPKP }: ConnectToVincentYieldModalProps) {
+  const [isRedirecting, setIsRedirecting] = useState(false);
+
+  const handleConnect = () => {
+    setIsRedirecting(true);
+    const redirectUri = encodeURIComponent('https://yield.heyvincent.ai');
+    const connectUrl = `${window.location.origin}/user/appId/2353287477/connect?redirectUri=${redirectUri}`;
```

Comment:
should we use the `env.VITE_VINCENT_YIELD_APPID` in the URL here?  so if we ever change the app id in the env var it updates here too?